castai/resource_node_template_test.go

```diff
@@ -264,7 +266,7 @@ func testAccNodeTemplateConfig(rName, clusterName string) string {
 }
 
 func testNodeTemplateUpdated(rName, clusterName string) string {
-	return ConfigCompose(testAccClusterConfig(rName, clusterName), testAccNodeConfig(rName), fmt.Sprintf(`
+	return ConfigCompose(testAccEKSClusterConfig(rName, clusterName), testAccNodeConfig(rName), fmt.Sprintf(`
```

Comment:
EKS?

Reply:
The name is correct. I've created a new `testAccClusterConfig` method, to not bring confusion I've added the prefix `AKS`. So now we have `testAccEKSClusterConfig` and `testAccAKSClusterConfig`

Reply:
got it, clear